packages/web-app-files/src/views/shares/SharedWithMe.vue

```diff
@@ -55,6 +56,7 @@ import { useGroupingSettings } from '@ownclouders/web-pkg'
 import SharesNavigation from 'web-app-files/src/components/AppBar/SharesNavigation.vue'
 import { useGettext } from 'vue3-gettext'
 import { useStore } from '@ownclouders/web-pkg'
+import { useOpenWithDefaultApp } from '../../composables/openWithDefaultApp'
```

Comment:
Good point, fixed here for the `useOpenWithDefaultApp` composable. We need to be careful with the imports still, sometimes unit tests require these specific imports to be able to mock stuff and to avoid circular imports.